structopt-derive/src/attrs.rs

```diff
@@ -95,8 +95,10 @@ impl Method {
             None => match env::var(env_var) {
                 Ok(val) => LitStr::new(&val, ident.span()),
                 Err(_) => {
-                    span_error!(ident.span(),
-                        "`{}` environment variable is not defined, use `{} = \"{}\"` to set it manually", env_var, ident, ident
+                    abort!(ident.span(),
+                        "cannot derive `{}` from Cargo.toml", ident;
+                        note = "`{}` environment variable is not set", env_var;
+                        help = "use `{} = \"{}\"` to set {} manually", ident, ident, ident;
```

Comment:
use `#[structopt({} = \"{}\")]` to set {} manually

Reply:
That's right. It'll be turned into "use `about = "about"`  to set about manually. Too much "about"?